### PR TITLE
ENT-5388: documenting backchainFetchBatchSize

### PR DESCRIPTION
**Pull request by AMcFarlandCampbellR3**

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.4/node/setup/corda-configuration-file.md [hunk 1]
```diff
@@ -191,6 +191,14 @@ The available configuration fields are listed below in alphabetic order.
 
   *Default:* 1024
 
+`backchainFetchBatchSize`
```
Comment (dimosr) [inline content/en/docs/corda-enterprise/4.4/node/setup/corda-configuration-file.md:194]: I think that should go under the `tuning` section below.

File: content/en/docs/corda-enterprise/4.4/node/setup/corda-configuration-file.md [hunk 2]
```diff
@@ -191,6 +191,14 @@ The available configuration fields are listed below in alphabetic order.
...
+
+This is an optimisation for sharing transaction backchains. Corda Enterprise nodes can request backchain items in bulk instead of one at a time. This field specifies the size of the batch. The value is just an integer indicating the maximum number of states that can be requested at a time during backchain resolution.
+
+Responding nodes (Enterprise or Open Source) running at platform version >= 6 will supply backchain items in bulk up to half of the network’s allowed maximum message size. The minimum is one item and items exceeding the limit are sent in subsequent batches. Nodes running on older platform versions will still supply backchain items one at a time.
+
```
Comment (dimosr) [inline content/en/docs/corda-enterprise/4.4/node/setup/corda-configuration-file.md:199]: Not sure what's the detail we want to go into in this page given it's just documenting the config option (not the feature itself), but I think the second part might be too much of a detail. Trying to come up with something more concise, I ended up thinking we should probably just remove this part (which is already in the release notes).

File: content/en/docs/corda-enterprise/4.4/node/setup/corda-configuration-file.md (hunk 1 above)
Comment (AMcFarlandCampbellR3) [reply content/en/docs/corda-enterprise/4.4/node/setup/corda-configuration-file.md]: OK, I have moved it.

File: content/en/docs/corda-enterprise/4.4/node/setup/corda-configuration-file.md (hunk 2 above)
Comment (AMcFarlandCampbellR3) [reply content/en/docs/corda-enterprise/4.4/node/setup/corda-configuration-file.md]: OK, I have removed it.

**Pull request merged**